spec/support/create_strategy_for_repository_pattern.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+class CreateStategyForRepositoryPattern
+  def association(runner)
+    runner.run
+  end
+
+  def result(evaluation)
```

Comment:
This might be obvious to others, but why is this the workflow? Maybe a comment that points to where the real behavior/callbacks are defined?